src/main/kotlin/no/nav/syfo/sykmeldingstatus/api/v1/SykmeldingBekreftAvvistApi.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.sykmeldingstatus.api.v1
+
+import io.ktor.application.call
+import io.ktor.auth.authentication
+import io.ktor.auth.jwt.JWTPrincipal
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import no.nav.syfo.metrics.BEKREFTET_AV_BRUKER_COUNTER
+import no.nav.syfo.sykmeldingstatus.SykmeldingStatusService
+
+fun Route.registrerSykmeldingBekreftAvvistApi(sykmeldingStatusService: SykmeldingStatusService) {
+    post("/api/v1/sykmeldinger/{sykmeldingid}/bekreftAvvist") {
+        val sykmeldingId = call.parameters["sykmeldingid"]!!
+        val token = call.request.headers["Authorization"]!!
+        val principal: JWTPrincipal = call.authentication.principal()!!
+        val fnr = principal.payload.subject
+
+        sykmeldingStatusService.registrerBekreftetAvvist(
+                sykmeldingId = sykmeldingId,
+                source = "user",
+                fnr = fnr,
+                token = token
+        )
+
+        BEKREFTET_AV_BRUKER_COUNTER.inc()
```

Comment:
Kan vurdere å lage egen metrikk for bekreftet avvist :)

Reply:
god ide 👍